vec120/src/main/java/com/foursoft/vecmodel/vec120/SchemaFactory.java

```diff
@@ -0,0 +1,46 @@
+/*-
+ * ========================LICENSE_START=================================
+ * vec120
+ * %%
+ * Copyright (C) 2020 - 2021 4Soft GmbH
+ * %%
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ * =========================LICENSE_END==================================
+ */
+package com.foursoft.vecmodel.vec120;
+
+import javax.xml.validation.Schema;
+
+public class SchemaFactory {
+
+    private static final String SCHEMA_PATH = "vec120/vec_1.2.1.RC-SNAPSHOT.xsd";

```

Comment:
shouldn´t this be 1.2.0.xsd? what is 1.2.1-RC? I would rather have another PR once 1.2.1 is oficialle released and add the schemas on top

Reply:
I have already discussed this with @beckerjohannes. The schema is already in the repository since the beginning. We still want to adjust this in a separate branch.